scripts/test

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+docker-compose down -v
```

Comment:
The file argument (`-f docker-compose-tests.yml`) needs adding here for the `down` to be specific to the tests, AFAIK.